README.md

```diff
@@ -1,2 +1,19 @@
-# media
-Drupal 8 version of the Drupal media module.
+# Media Module
+
+A complete media module for Drupal 8.
```

Comment:
Since we work on the Media namespace we should have basically a very similar description as the module had in the D7 version. Take a look at https://www.drupal.org/node/356802 and https://www.drupal.org/project/file_browser.
Also I wouldn't emphasize D8 so much. It's already evident from the module version.